Themes/default/css/index.css

```diff
@@ -735,6 +740,9 @@ img.sort, .sort {
 #main_menu {
 	margin: 0 0 4px 0;
 }
+#main_menu .popup_window, #main_menu .popup_container {
+	z-index: unset;
```

Comment:
This doesn't quiet fix it though :p, the point of having the z-index on popup container is because on mobile it's needed, just not needed on desktop, so just removing it from line and adding it to responsive.css would fix it

But I just remembered you probably don't want to edit responsive.css right now :/ so ..

Reply:
Yeah, was avoiding editing responsive.css.

Hmm, it looked ok on my testing. I'll give it a better look. And since Sesq is merging the responsive PR soon so we can get back to meddling with it :)

Reply:
Checked again and still looks ok. You use chrome, right? It normally has a different idea of how to work with z-index.

Reply:
This is how it looks on my site.
![screenshot_2019-02-28 start new topic](https://user-images.githubusercontent.com/3620111/53605751-7bc9f400-3b86-11e9-9f87-e34376069060.png)

Reply:
What browser/version is that? I just tested on chrome and it also looks ok. But I see you have not added the latest changes of this PR, the popup should be small as well.

Reply:
Well it looks okay on desktop but not on mobile, here's what I mean
![onmob](https://user-images.githubusercontent.com/20267363/53639507-b5a00680-3c29-11e9-916a-35844d3ade5f.png)
Best way of fixing this is to remove **z-index: 99;** from **.pupup_window** and **_z-index: 5;_** from **_.popup_container_** in **index.css**, removing those doesn't affect the mobile view since those values are already set in **responsive.css**

and of course reverting the css change above

Reply:
Firefox65 64 bit